x-pack/plugin/src/test/resources/rest-api-spec/test/license/20_put_license.yml

```diff
@@ -133,7 +133,8 @@ teardown:
 
   - do:
       catch: forbidden
-      xpack.license.post_start_trial: {}
+      xpack.license.post_start_trial:
```

Comment:
should it still be post_start_trial and not changed to get_start_trial? Or just start_trial

Reply:
No it should be post_start_trial. We're testing that this call fails.